src/metabase/api/common.clj

```diff
@@ -263,7 +309,11 @@
                     assoc
                     :doc          docstr
                     :is-endpoint? true)
-     (~(symbol "compojure.core" (name method)) ~route ~args
+     (~(symbol "compojure.core" (name method))
```

Comment:
Seeing how we qualify the method in the macro anyway maybe we should update the kondo linter so it stops complaining if you don't explicitly `:refer` `GET` or `POST` or whatever

Reply:
Yeah that struck me. The macro just cares that they are the symbols `POST` or `GET`, etc. and not that they are actually vars.